cmd/osv-scanner/fixtures/locks-many/osv-scanner.toml

```diff
@@ -6,3 +6,7 @@ reason = "Test manifest file"
 [[IgnoredVulns]]
 id = "CVE-2022-48174"
 reason = "Test manifest file (alpine.cdx.xml)"
+
+[[IgnoredVulns]]
+id = "CVE-2025-26519"
```

Comment:
I think ignoring this is the right call, at least for now, because we're using this fixture in a couple of different tests including some that look like they're expecting no vulnerabilities

(if I could go back in time, I think I'd push for us to include some form of readme in each of these fixtures to capture what the fixture is trying to provide)